lib/ueberauth/strategy/passwordless.ex

```diff
@@ -77,11 +77,17 @@ defmodule Ueberauth.Strategy.Passwordless do
   use Ueberauth.Strategy
 
   alias Ueberauth.Auth.{Extra, Info}
+  alias Ueberauth.Strategy.Passwordless.Store
 
   @defaults [
-    # Default TTL is 60 Minutes.
-    ttl: 60 * 60,
```

Comment:
es muss eine stunde sein

Reply:
Den Standardwert welchen ich bzg. Passwordless gesehen habe lag immer bei 15min. Daher würde ich ganz gerne den Standardwert von 15min in die Library packen und diesen dann in Einsatzplanung in der `config.exs` mit 60min überschreiben